package/src/types.ts

```diff
@@ -74,6 +74,11 @@ export interface BaseGuardProps {
 
 export type PropsWithMeta<T> = T & {
   meta?: Meta;
+  pathChanged?: (
```

Comment:
Instead of adding the type here, we'll want to add it to the GuarderRoute and GuardProps below